settings/base.py

```diff
@@ -0,0 +1,160 @@
+"""
+Base settings for ORA2.
```

Comment:
I think this setting file is only used when developing under workbench, not being used when deployed in edx. I would expect this XBlock to be installed as an XBlock.